alpha/scripts/utils/enableAdminConsoleSso.php

```diff
@@ -0,0 +1,41 @@
+<?php
+if ($argc < 2)
+{
+	die ($argv[0]. " <redirect_url> <domain> \n");
+}
+
+$redirectUrl = $argv[1];
+$domain = null;
+if($argc == 3)
+{
+	$domain = $argv[2];
+}
+
+require_once(__DIR__ . '/../bootstrap.php');
+const ADMIN_CONSOLE_APPLICATION_TYPE = 'admin_console';
+
+$partner = PartnerPeer::retrieveActiveByPK(Partner::ADMIN_CONSOLE_PARTNER_ID);
+if($partner)
+{
+	allowSsoFeature($partner);
+	addSSoProfile($domain, $redirectUrl);
+}
+
+function allowSsoFeature($partner)
+{
+	$partner->setUseSso(true);
+	$partner->setBlockDirectLogin(true);
+	$partner->save();
+}
+
+function addSsoProfile($domain, $redirectUrl)
```

Comment:
What about the SSO data used for the authentication, how do you add it to the profile?